src/web/templates/member/screen-o-death.html

```diff
@@ -1,5 +1,7 @@
 {% include 'base-header.html' %}
 
+<script src="//cdnjs.cloudflare.com/ajax/libs/bootbox.js/4.3.0/bootbox.min.js"></script>
```

Comment:
@Sanrag can this be loaded at the bottom of the page? If so you can add it into `screen-o-death.js`